src/pages/student/student.jsx

```diff
@@ -144,13 +151,17 @@ const StudentView = () => {
               6: '6th Grade',
             },
             ethnicityOptions: [
-              'white',
-              'black',
-              'asian',
-              'latinx',
-              'american indian or alaska native',
-              'non-specified',
+              { value: 'white', label: 'white' },
+              { value: 'black', label: 'black' },
+              { value: 'asian', label: 'asian' },
+              { value: 'latinx', label: 'latinx' },
+              {
+                value: 'american indian or alaska native',
+                label: 'american indian or alaska native',
+              },
+              { value: 'non-specified', label: 'non-specified' },
             ],
+            genderOptions: ['male', 'female', 'non-specified'],
             studentGroups: [...resOptions.data],
```

Comment:
Check [this thread](https://ctcuci.slack.com/archives/C02JMK343V4/p1652308837091989) for whether or not to refine this data more